python/Ganga/GPIDev/Lib/Job/JobTime.py

```diff
@@ -66,6 +66,7 @@ class JobTime(GangaObject):
     _category = 'jobtime'
     _name = 'JobTime'
     _exportmethods = ['display',
+                      '_display',
```

Comment:
Does `_display` need to be exported? I don't think we want end-users calling the function.